datafusion/sql/src/statement.rs

```diff
@@ -706,13 +713,18 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
     /// Generate a plan for EXPLAIN ... that will print out a plan
     ///
-    fn explain_statement_to_plan(
+    /// Note this is the sqlparser explain statement, not the
+    /// datafusion `EXPLAIN` statement.
+    fn explain_to_plan(
         &self,
         verbose: bool,
         analyze: bool,
-        statement: Statement,
+        statement: DFStatement,
```

Comment:
Can we use `ExplainStatement` as param?

Reply:
So there are two places this is called -- one from the DataFusion `ExplainStatement` (introduced in this PR) and one from the SQL parser [`Explain`](https://docs.rs/sqlparser/0.36.1/sqlparser/ast/enum.Statement.html#variant.Explain). So I thought it best to support them both and simply pass the parameters on through that are actually used